exec/java-exec/src/main/codegen/data/Parser.tdd

```diff
@@ -118,780 +118,824 @@
-        "RELATIVE"
-        "REPEATABLE"
-        "REPLACE"
-        "RESTART"
-        "RESTRICT"
-        "RETURNED_CARDINALITY"
-        "RETURNED_LENGTH"
-        "RETURNED_OCTET_LENGTH"
-        "RETURNED_SQLSTATE"
-        "ROLE"
-        "ROUTINE"
-        "ROUTINE_CATALOG"
-        "ROUTINE_NAME"
-        "ROUTINE_SCHEMA"
-        "ROW_COUNT"
-        "SCALE"
-        "SCHEMA"
-        "SCHEMA_NAME"
-        "SCOPE_CATALOGS"
-        "SCOPE_NAME"
-        "SCOPE_SCHEMA"
-        "SECTION"
-        "SECURITY"
-        "SELF"
-        "SEQUENCE"
-        "SERIALIZABLE"
-        "SERVER"
-        "SERVER_NAME"
-        "SESSION"
-        "SETS"
-        "SIMPLE"
-        "SIZE"
-        "SOURCE"
-        "SPACE"
-        "SPECIFIC_NAME"
-        "SQL_BIGINT"
-        "SQL_BINARY"
-        "SQL_BIT"
-        "SQL_BLOB"
-        "SQL_BOOLEAN"
-        "SQL_CHAR"
-        "SQL_CLOB"
-        "SQL_DATE"
-        "SQL_DECIMAL"
-        "SQL_DOUBLE"
-        "SQL_FLOAT"
-        "SQL_INTEGER"
-        "SQL_INTERVAL_DAY"
-        "SQL_INTERVAL_DAY_TO_HOUR"
-        "SQL_INTERVAL_DAY_TO_MINUTE"
-        "SQL_INTERVAL_DAY_TO_SECOND"
-        "SQL_INTERVAL_HOUR"
-        "SQL_INTERVAL_HOUR_TO_MINUTE"
-        "SQL_INTERVAL_HOUR_TO_SECOND"
-        "SQL_INTERVAL_MINUTE"
-        "SQL_INTERVAL_MINUTE_TO_SECOND"
-        "SQL_INTERVAL_MONTH"
-        "SQL_INTERVAL_SECOND"
-        "SQL_INTERVAL_YEAR"
-        "SQL_INTERVAL_YEAR_TO_MONTH"
-        "SQL_LONGVARBINARY"
-        "SQL_LONGVARNCHAR"
-        "SQL_LONGVARCHAR"
-        "SQL_NCHAR"
-        "SQL_NCLOB"
-        "SQL_NUMERIC"
-        "SQL_NVARCHAR"
-        "SQL_REAL"
-        "SQL_SMALLINT"
-        "SQL_TIME"
-        "SQL_TIMESTAMP"
-        "SQL_TINYINT"
-        "SQL_TSI_DAY"
-        "SQL_TSI_FRAC_SECOND"
-        "SQL_TSI_HOUR"
-        "SQL_TSI_MICROSECOND"
-        "SQL_TSI_MINUTE"
-        "SQL_TSI_MONTH"
-        "SQL_TSI_QUARTER"
-        "SQL_TSI_SECOND"
-        "SQL_TSI_WEEK"
-        "SQL_TSI_YEAR"
-        "SQL_VARBINARY"
-        "SQL_VARCHAR"
-        "STATE"
-        "STATEMENT"
-        "STORAGE"
-        "STRUCTURE"
-        "STYLE"
-        "SUBCLASS_ORIGIN"
-        "SUBSTITUTE"
-        "TABLE_NAME"
-        "TEMPORARY"
-        "TIES"
-        "TIMESTAMPADD"
-        "TIMESTAMPDIFF"
-        "TOP_LEVEL_COUNT"
-        "TRANSACTION"
-        "TRANSACTIONS_ACTIVE"
-        "TRANSACTIONS_COMMITTED"
-        "TRANSACTIONS_ROLLED_BACK"
-        "TRANSFORM"
-        "TRANSFORMS"
-        "TRIGGER_CATALOG"
-        "TRIGGER_NAME"
-        "TRIGGER_SCHEMA"
-        "TYPE"
-        "UNBOUNDED"
-        "UNCOMMITTED"
-        "UNDER"
-        "UNNAMED"
-        "USAGE"
-        "USER_DEFINED_TYPE_CATALOG"
-        "USER_DEFINED_TYPE_CODE"
-        "USER_DEFINED_TYPE_NAME"
-        "USER_DEFINED_TYPE_SCHEMA"
-        "VERSION"
-        "VIEW"
-        "WEEK"
-        "WRAPPER"
-        "WORK"
-        "WRITE"
-        "XML"
-        "ZONE"
+  nonReservedKeywords: [
```

Comment:
Do the updates to these lists comes from some corresponding lists that were updated in Calcite?

Reply:
Yes, as mentioned in several rows above, they are copied from calcite-core `config.fmpp`. We use our own freemarker config file, so we can't inherit lists they have defined.